src/ServiceControl.Plugin.Nsb6.SagaAudit/SagaAudit.cs

```diff
@@ -13,13 +15,26 @@ public SagaAudit()
             EnableByDefault();
             DependsOn<Sagas>();
         }
-        
+
         protected override void Setup(FeatureConfigurationContext context)
         {
             context.Container.ConfigureComponent<ServiceControlBackend>(DependencyLifecycle.SingleInstance);
 
             context.Pipeline.Register<CaptureSagaStateRegistration>();
             context.Pipeline.Register<CaptureSagaResultingMessageRegistration>();
```

Comment:
Mike:

This is outdated. I am just working acceptance tests now that the core pr
was merged.

On Sun, Mar 20, 2016, 9:41 PM Mike Minutillo <notifications@github.com>
wrote:

> In src/ServiceControl.Plugin.Nsb6.SagaAudit/SagaAudit.cs
> <https://github.com/Particular/ServiceControl.Plugin.Nsb6.SagaAudit/pull/2#discussion_r56782067>
> :
>
> > @@ -13,13 +15,26 @@ public SagaAudit()
> >              EnableByDefault();
> >              DependsOn<Sagas>();
> >          }
> > -
> > +
> >          protected override void Setup(FeatureConfigurationContext context)
> >          {
> >              context.Container.ConfigureComponent<ServiceControlBackend>(DependencyLifecycle.SingleInstance);
> >
> >              context.Pipeline.Register<CaptureSagaStateRegistration>();
> >              context.Pipeline.Register<CaptureSagaResultingMessageRegistration>();
>
> This behavior is in the outgoing context which does not include the
> InvokeSaga step. It does not need the InsertAfter(..) call as it will
> always take place after the saga has started (or if there was no saga)
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/ServiceControl.Plugin.Nsb6.SagaAudit/pull/2/files/6ef188210ec18800e8bbe5600e6868c95cc45568#r56782067>
>